src/gossip_agent/waku_handling.rs

```diff
@@ -247,9 +248,11 @@ pub fn setup_node_handle(
             boot_node_handle
         }
         _ => {
-            let mut file = File::open("./boot_node_addr.conf").unwrap();
+            let mut file =
+                File::open("./boot_node_addr.conf").expect("Could not find boot_node_addr.conf");
```

Comment:
btw I don't think a light node should panic if this file is not found, because by default it should try to connect to a full node. But I'll add a check for that in the next PR since I already have a draft of it going, just wanted to get your thoughts

Reply:
honestly, I think we should remove read/write from files altogether since now the node id and multi-addresses should be consistent across instances

Reply:
agreed!

Reply:
Does this mean we should remove the static node option entirely?

Reply:
nope I think it can be an environmental variable. static list would be great for fallback